src/cw_std.rs

```diff
@@ -0,0 +1,62 @@
+// no_std support
```

Comment:
But do we really need this file? I'm pretty sure we can just use the `alloc` and `core` crates directly, even in `std` environment. `std` just re-exports them anyway.

Reply:
We want to conditionally use them.
I wouldn't use `core` libs in `std`. It gets all kinds of confusion when you start passing `core::Vec` to a lib that expects `std::Vec`

If we want to support both, such a set of conditional imports seems a proper way to do it.

Reply:
You mean "confusion" as in: The crate user will get confused? Just asking because `alloc::vec::Vec` and `std::vec::Vec` are just two names for the same thing, so passing them around works perfectly fine.

But if you prefer importing it from std, then yes, the conditional imports file is the best solution.